src/BasePlatform.js

```diff
@@ -38,4 +38,21 @@ export default class BasePlatform {
 
     displayNotification(title: string, msg: string, avatarUrl: string) {
     }
+
+    /**
+     * Check for the availability of an update to the version of the
+     * app that's currently running.
+     * If an update is available, this function should dispatch the
+     * 'new_version' action.
+     */
+    pollForUpdate() {
```

Comment:
given that these aren't used in the react-sdk layer, I'm not sure how I feel about having them here. Maybe we should have a VectorBasePlatform instead, or something?